### DISCUSSION TITLE
A1841CBA-C50B-51C7-6AF2-AC6DFB184F9D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#100] [General]
# Black Hole System

A super massive black hole surrounded by a ring of eight stars and 550 habital planets with varying atmospheres and surface textures.

https://celestia.mobi/addon?item=A1841CBA-C50B-51C7-6AF2-AC6DFB184F9D

**Comment by egg298** [comment]
Wow. Very cool place

**Comment by Kybeau** [comment]
IWANTIt!!!!!!!

**Comment by Ledeledeli** [comment]
What’s it called in the Celestia search?

**Comment by llyaf4444-maker** [comment]
Очень прикольные планеты с жизнью но их 550 где то но всё равно хорошее место где много таких планет
